skrub/_table_vectorizer.py

```diff
@@ -115,38 +115,31 @@ class TableVectorizer(TransformerMixin, BaseEstimator):
 
     Applies a different transformation to each of several kinds of columns:
 
-    - numeric:
-        Floats, ints, and Booleans.
-    - datetime:
-        Datetimes and dates.
-    - low_cardinality:
-        String and categorical columns with a count of unique values smaller
-        than a given threshold (40 by default). Category encoding schemes such
-        as one-hot encoding, ordinal encoding etc. are typically appropriate
-        for columns with few unique values.
-    - high_cardinality:
-        String and categorical columns with many unique values, such as
-        free-form text. Such columns have so many distinct values that it is
-        not possible to assign a distinct representation to each: the dimension
-        would be too large and there would be too few examples of each
-        category. Representations designed for text, such as topic modelling
-        (GapEncoder) or locality-sensitive hashing (MinHash) are more
-        appropriate.
+    - numerical features: floats, integers, and booleans.
```

Comment:
to avoid having to remember for every parameter, attribute or variable whether to type "numeric" or "numerical" we decided to only ever use "numeric" (they are synonyms)

Reply:
Also I would prefer to talk about "columns" for the inputs rather than "features" . In the context of a dataframe I think it is more obvious what is a "column" than a "feature", also before the tablevectorizer numeric features haven't been extracted yet.